### PR TITLE
#167769453 Enable LandVille Admin to manage terms of Use

### PR DESCRIPTION
**Pull request by CryceTruly**
## Description ##
Enable LandVille Admin to manage terms of Use

## Type of change ##
Please select the relevant option
- [ ] Bug fix(a non-breaking change which fixes an issue)
- [x] New feature(a non-breaking change which adds functionality)
- [ ] Breaking change(fix of a feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested? ##
Please describe the tests that you ran to verify your changes
- [ ] End to end test
- [ ] Integration test
- [x] unit test

## Checklist: ##
- [x] My code follows the style guidelines of this project
- [x] I have linted my code prior to submission
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing tests pass locally with my changes

## How to test: ##
- Clone the repo
- Activate your virtual environment 
- Install dependencies using pip install -r requirements.txt
- Checkout branch ft-add-terms-of-service-167769453
- Setup Postgres and run migrations
- Create a superuser and log him in and navigate to the registered Terms model.

## Related Pivotal Tracker stories ##
[#167769453](https://www.pivotaltracker.com/story/show/167769453)

### CODE REVIEW

File: pages/migrations/0002_auto_20190812_1244.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    dependencies = [
+        ('pages', '0001_initial'),
+    ]
+
+    def insertData(apps, schema_editor):
+        Term = apps.get_model('pages', 'Term')
+        term = Term(details="Edit this")
+        term.save()
+    operations = [
+        migrations.RunPython(insertData),
```
Comment (patrickf949) [inline pages/migrations/0002_auto_20190812_1244.py:17]: Legendary 😎

**Comment by alexxsanya** [comment]
@CryceTruly, Welldone on this though I am unable to test out the functionality implemented via the deployment as I am getting a page with no deployed app as you may see below. Kindly guide me on how I can test this manually.

Screenshot 2019-08-14 at 10 42 15

**Comment by CryceTruly** [comment]
> @CryceTruly, Welldone on this though I am unable to test out the functionality implemented via the deployment as I am getting a page with no deployed app as you may see below. Kindly guide me on how I can test this manually.
> 
> Screenshot 2019-08-14 at 10 42 15

Thanks, @alexxsanya,
Now what is happening is we have a problem with review apps development on heroku.
So what we do is clone and test the branch locally,its some setup but that is how we still do it.
Maybe we can set up a sync and I walk you through.

File: pages/views.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+from rest_framework.response import Response
+from rest_framework import status
+
+
+class TermsView(APIView):
+    """Handles sending terms and conditions to users."""
+    serializer_class = PagesSerializer
+
+    def get(self, request):
+        terms = Term.objects.latest('last_updated_at')
```
Comment (Genza999) [inline pages/views.py:15]: Hey @CryceTruly , great work, could you kindly add a docstring here describing what this get method does.

Comment (CryceTruly) [reply pages/views.py:15]: Thanks, @Genza999, I have worked on this.

**Review by Genza999 — APPROVED** [review]
LGTM

**Review by dorothyas — APPROVED** [review]

**Review by shemaeric — APPROVED** [review]
LGTM

**Review by Victoradukwu — APPROVED** [review]
LGTM

**Review by JamesMudidi — APPROVED** [review]
LGTM

**Pull request merged**